apps/catalog/catalog/src/app/marketplace/offer/shell/contract-view/contract-view.component.html

```diff
@@ -1,10 +1,31 @@
 <a mat-icon-button routerLink="..">
   <mat-icon svgIcon="arrow_back"></mat-icon>
 </a>
-<ng-container *ngIf="contract$ | async as contract">
-  <contract-item [contract]="contract">
+<ng-container *ngIf="contract$ | async as sale">
+  <ng-container *ngIf="sale.negotiation?.status === 'pending' && sale.negotiation?.createdByOrg !== activeOrgId">
+    <div fxLayout="row" fxLayoutAlign="center center" fxLayoutGap="24px">
```

Comment:
use header, no need to specify `row` as it's the default value
```suggestion
    <header fxLayout fxLayoutAlign="center center" fxLayoutGap="24px">
```